Build.ps1

```diff
@@ -54,14 +30,8 @@ Process {
     # Pack using NuGet.exe
     Exec { & nuget pack Workleap.DotNet.CodingStandards.nuspec -OutputDirectory $outputDir -Version $version -ForceEnglishOutput }
 
-    # Create a new test console project, add our newly created package and try to build it in release mode
-    # The default .NET console project template with top-level statements should not trigger any warnings
-    # We treat warnings as errors even though it's supposed to be already enabled by our package,
-    # just in case the package is not working as expected
-    Exec { & dotnet new console --name $testProjectName --output $testProjectDir }
-    Set-Content -Path $testNugetConfigPath -Value $testNugetConfigContents
-    Exec { & dotnet add $testProjectPath package Workleap.DotNet.CodingStandards --version $version }
-    Exec { & dotnet build $testProjectPath --configuration Release /p:TreatWarningsAsErrors=true }
+    # Run tests
+    Exec { & dotnet test }
```

Comment:
```suggestion
    Exec { & dotnet test -c Release -l "console;verbosity=detailed" }
```

- `-c Release` because we may want our coding standards on this project as well and fail the build if any C# tooling or test code isn't compliant
- `-l "console;verbosity=detailed"` so we can see the xunit logger output in GitHub actions console (right now we can only see the summary of X tests passed/failed/etc.)